src/common/constants/index.ts

```diff
@@ -0,0 +1,49 @@
+export const CATAGORY: string[] = ["한식", "중식", "일식", "양식", "분식", "카페", "편의점", "기타"];
+
+export const ADRESS: string[] = [
+  "서울시 종로구",
+  "서울시 중구",
+  "서울시 용산구",
+  "서울시 성동구",
+  "서울시 광진구",
+  "서울시 동대문구",
+  "서울시 중랑구",
+  "서울시 성북구",
+  "서울시 강북구",
+  "서울시 도봉구",
+  "서울시 노원구",
+  "서울시 은평구",
+  "서울시 서대문구",
+  "서울시 마포구",
+  "서울시 양천구",
+  "서울시 강서구",
+  "서울시 구로구",
+  "서울시 금천구",
+  "서울시 영등포구",
+  "서울시 동작구",
+  "서울시 관악구",
+  "서울시 서초구",
+  "서울시 강남구",
+  "서울시 송파구",
+  "서울시 강동구",
+];
+
+export const PLACEHOLDER: { [key: string]: string } = {
+  SEARCH: "가게 이름으로 찾아보세요",
+  EMAIL: "이메일을 입력해주세요",
+  PASSWORD: "비밀번호를 8자리 이상으로 입력해주세요",
+  NAME: "식당 이름을 입력해주세요",
+  ADDRESS2: "상세주소를 입력해주세요",
```

Comment:
이렇게 표현해주는게 더 좋지 않을까요?
```suggestion
  DETAIL_ADDRESS: "상세주소를 입력해주세요",
```

Reply:
api에서 address2로 값을 사용하길래 동일하게 작성했는데 수정하는게 좋을 것 같네요